.github/workflows/review_cypress_e2e_tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Cypress end-to-end tests
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-16.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          browser: chrome
+          working-directory: cypresse2e
+          start: npx cypress run
```

Comment:
interesting... didn't know Github actions use `npx` ... I guess this always automatically uses the latest version